### DISCUSSION TITLE
火狐浏览器还发版吗？

### DISCUSSION THREAD

**Discussion by yuyuko233** [crimx/ext-saladict#2061] [Q&A]
火狐浏览器还发版吗？我看最后一版怎么还是7.11呢

**Comment by crimx** [comment]
火狐突然换了个人审核，原来通过的现在又说不行，最近几个版本都下架了，要改的比较多。

**Reply by yuyuko233** [reply]
那现在有大概的发版计划吗

**Comment by echosori** [comment]
请问火狐版本如何安装？ 我下载了Firefox.zip，也改了火狐设置，但下载的压缩文件中没有XPI文件。
![image](https://github.com/crimx/ext-saladict/assets/19378581/9f8cb940-1348-484a-ba54-7416c1eddf4f)
